cmake/apple/install_apple.sh.in

```diff
@@ -21,20 +21,16 @@ MSG_COLOR='\x1B[1;36m'
 COLOR_RESET='\x1B[0m'
 echo -e "$MSG_COLOR\n\nCreating App Bundle \"$APP\"...$COLOR_RESET"
 
-# Locate macdeployqt, assume homebrew & Qt5
-which macdeployqt > /dev/null 2>&1
```

Comment:
I don't have a clear idea of what MacPorts is. But I didn't saw the purpose of this command, it's just to locate the `macdeployqt` binary, then, if it's not here, we don't add QT path to `$PATH` but we still call `macdeployqt` on line 96.